pkg/fanal/secret/builtin-rules.go

```diff
@@ -743,6 +744,24 @@ var builtinRules = []Rule{
 		Regex:    MustCompile(`pscale_tkn_(?i)[a-z0-9\-_\.]{43}`),
 		Keywords: []string{"pscale_tkn_"},
 	},
+	{
+		ID:       "private-packagist-user-token",
+		Category: CategoryPrivatePackagist,
+		Title:    "Private Packagist user token",
+		Severity: "HIGH",
+		// https://packagist.com/docs/composer-authentication#token-format
```

Comment:
I don't see any other comments referring to the documentation of the token format, but I thought it would be useful.
Let me know if I should remove it.